src/pages/PostWritePage.jsx

```diff
@@ -20,37 +43,62 @@ const PostWritePage = () => {
     }
   };
 
+  const handleAlert = () => {
+    Swal.fire({
+      title: '공고를 등록하시겠습니까?',
+      html: `정보를 알맞게 입력하셨나요?<br/> 
+      피커는 입력한 정보를 바탕으로 움직이게 됩니다.`,
+      icon: 'question',
+      showCancelButton: true,
+      reverseButtons: true,
+      confirmButtonColor: '#0075FF',
+      cancelButtonColor: '#D9D9D9',
+      cancelButtonText: '취소',
+      confirmButtonText: '확인',
+    });
+  };
+
   const handleNext = () => {
-    if (focus < 3) {
+    if (focus === 1 && orderInfoValue[0] && orderInfoValue[1]) {
       setFocus((prev) => prev + 1);
     }
+    if (focus === 2 && requestValue[0]) {
+      setFocus((prev) => prev + 1);
+    }
+    if (focus === 3) handleAlert();
+  };
+
+  const onSubmit = (data) => {
+    console.log(data); // api 연결 후 수정 예정
   };
 
   switch (focus) {
```

Comment:
IIFE로 감싸는게 좋을거 같아요
코드 자체가 분리가 안되는것 같아요
```suggestion
const page = (function() { return .... })()
```